e2e/playwright/list-views/src/tests/sort-list.spec.ts

```diff
@@ -0,0 +1,291 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import {
+  ApiClientFactory,
+  FavoritesPageApi,
+  FileActionsApi,
+  LoginPage,
+  NodesApi,
+  TEST_FILES,
+  Utils,
+  test,
+  timeouts
+} from '@alfresco/playwright-shared';
+
+test.describe('Remember sorting', () => {
```

Comment:
Is it possible to run those tests together with ones related to login in the upper file? The setup is quite similar and in this way we can reuse some setup code

Reply:
It was getting some failure due to logout related test,  now tries some approach and add code in one file.